docs/source/routing/performance/query-batching.mdx

```diff
@@ -1,16 +1,24 @@
 ---
 title: Query Batching
 subtitle: Receive query batches with the GraphOS Router
-description: Handle multiple GraphQL requests with GraphOS Router's query batching capabilities. Aggregate operations into single HTTP requests, reducing overhead.
+description: Handle multiple GraphQL requests with GraphOS Router's query batching capabilities. Aggregate operations into single HTTP requests, preserving data consistency and reducing performance overhead.
```

Comment:
I think we should get rid of "performance overhead". Batching shouldn't be used for perf optimisations, but rather for data consistency. In fact, most of the times batching operations is going to negatively affect performance, but if you're doing this for data consistency purposes, you are usually okay with the performance hit.

Reply:
```suggestion
description: Handle multiple GraphQL requests with GraphOS Router's query batching capabilities. Aggregate operations into single HTTP requests to preserve data consistency.
```

Reply:
ohhh yess!